test.js

```diff
@@ -10,3 +10,43 @@ test(t => {
 	t.is(m.getName(143), 'Snorlax');
 	t.is(m.getId('Snorlax'), 143);
 });
+
+test('English names', t => {
+	t.is(m.getName(1), 'Bulbasaur');
+	t.is(m.getName(100), 'Voltorb');
+	t.is(m.getName(200), 'Misdreavus');
+	t.is(m.getName(300), 'Skitty');
+	t.is(m.getName(400), 'Bibarel');
+	t.is(m.getName(500), 'Emboar');
+	t.is(m.getName(600), 'Klang');
+	t.is(m.getName(700), 'Sylveon');
+	t.is(m.getName(721), 'Volcanion');
+});
+
+test('German names', t => {
+	t.is(m.getName(1, 'de'), 'Bisasam');
+	t.is(m.getName(100, 'de'), 'Voltobal');
+	t.is(m.getName(200, 'de'), 'Traunfugil');
+	t.is(m.getName(300, 'de'), 'Eneco');
+	t.is(m.getName(400, 'de'), 'Bidifas');
+	t.is(m.getName(500, 'de'), 'Flambirex');
+	t.is(m.getName(600, 'de'), 'Kliklak');
+	t.is(m.getName(700, 'de'), 'Feelinara');
+	t.is(m.getName(721, 'de'), 'Volcanion');
+});
```

Comment:
These two tests are pretty much identical except for the names. Can you DRY it up a bit in a [test macro](https://github.com/avajs/ava#test-macros)? Don't need to test that many names either. It's enough with 4.